diff -up tclvfs-20080503/generic/vfs.c.tcl86 tclvfs-20080503/generic/vfs.c
--- tclvfs-20080503/generic/vfs.c.tcl86	2014-06-10 13:49:28.758558734 -0400
+++ tclvfs-20080503/generic/vfs.c	2014-06-10 13:49:42.673469640 -0400
@@ -1658,7 +1658,7 @@ VfsRemoveDirectory(
 
 static Tcl_FSFileAttrStringsProc VfsFileAttrStrings;
 
-static CONST char**
+static const char *CONST86 *
 VfsFileAttrStrings(pathPtr, objPtrRef)
     Tcl_Obj* pathPtr;
     Tcl_Obj** objPtrRef;
