diff -up tclvfs-20080503/generic/vfs.c.tcl86 tclvfs-20080503/generic/vfs.c
--- tclvfs-20080503/generic/vfs.c.tcl86	2014-06-10 14:06:59.568275464 -0400
+++ tclvfs-20080503/generic/vfs.c	2014-06-10 14:07:54.950900301 -0400
@@ -1656,7 +1656,7 @@ VfsRemoveDirectory(
     return returnVal;
 }
 
-static CONST char**
+static const char *CONST86 *
 VfsFileAttrStrings(pathPtr, objPtrRef)
     Tcl_Obj* pathPtr;
     Tcl_Obj** objPtrRef;
